cloud-consumption/src/main/java/com/sequenceiq/consumption/dto/StorageConsumptionResult.java

```diff
@@ -1,10 +1,14 @@
 package com.sequenceiq.consumption.dto;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class StorageConsumptionResult {
 
     private final double storageInBytes;
 
-    public StorageConsumptionResult(double storageInBytes) {
+    @JsonCreator
```

Comment:
just trying to understand the change, why is this added to this dto class and not to the other ones as well? (ConsumptionCreationDto, Credential)

Reply:
because it is referenced from a flow payload class (most probably one its fields is this type)